src/components/App.js

```diff
@@ -84,62 +82,90 @@ var App = React.createClass({
     NetworkActions.leaveChannel.listen(this.onLeaveChannel)
     AppActions.login.listen(this.onLogin)
 
-    this.unsubscribeFromNetworkStore = NetworkStore.listen(this.onNetworkUpdated)
+    this.unsubscribeFromNetworkStore = NetworkStore.listen(
+      this.onNetworkUpdated
+    )
     this.unsubscribeFromUserStore = UserStore.listen(this.onUserUpdated)
-    this.stopListeningAppState = AppStateStore.listen(this._handleAppStateChange)
-    this.unsubscribeFromSettingsStore = SettingsStore.listen((settings) => {
-      this.setState({ theme: Themes[settings.theme] || null, leftSidePanel: settings.leftSidePanel })
+    this.stopListeningAppState = AppStateStore.listen(
+      this._handleAppStateChange
+    )
+    this.unsubscribeFromSettingsStore = SettingsStore.listen(settings => {
+      this.setState({
+        theme: Themes[settings.theme] || null,
+        leftSidePanel: settings.leftSidePanel
+      })
     })
 
     // window.onblur = () => AppActions.windowLostFocus()
     // window.onfocus = () => AppActions.windowOnFocus()
   },
-  _handleAppStateChange: function(state) {
-    let prefix = '', suffix = ''
+  _handleAppStateChange: function (state) {
+    let prefix = ''
 
-    if(!AppStateStore.state.hasFocus && AppStateStore.state.unreadMessages[AppStateStore.state.currentChannel] > 0)
-      suffix = `(${AppStateStore.state.unreadMessages[AppStateStore.state.currentChannel]})`
+    let suffix = ''
 
-    if(Object.keys(state.unreadMessages).length > 1 || (Object.keys(state.unreadMessages).length === 1 && !Object.keys(state.unreadMessages).includes(AppStateStore.state.currentChannel)))
+    if (
+      !AppStateStore.state.hasFocus &&
```

Comment:
Is the line break here due to linter? I personally find that a little harder to read, but if that's what the linter says, then let's go with that.